src/jubilation-line/index.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import getContext from '../helpers/context';
+import Animation from '../jubilation-animation';
+
+type Props = { x1?: number, x2?: number, y1?: number, y2?: number};
```

Comment:
Actually that a good point. Why don't you choose some reasonable defaults for the line coordinates similar to what we are doing in `Point`